pubspec.yaml

```diff
@@ -27,6 +27,11 @@ dependencies:
   firebase_core: ^1.6.0
   firebase_analytics: ^8.3.1
   firebase_auth: ^3.1.0
+  
+  font_awesome_flutter: ^9.1.0
+  flutter_svg: ^0.23.0+1
+  google_fonts: ^2.1.0
+  carousel_slider: ^4.0.0
```

Comment:
Why do you need two packages for fonts. `google fonts` and `font awesoem flutter`

Reply:
I added font_awesome_flutter to get some additional icons that conventionally flutter doesn't provide. It's of no use in my PR but it may be useful to include some icons in the future with ease.

Reply:
Acknowledged